aws/resource_aws_vpc_peering_connection_test.go

```diff
@@ -509,6 +509,7 @@ func testAccCheckAWSVpcPeeringConnectionOptionsWithProvider(n, block string, opt
 }
 
 func TestAccAWSVPCPeeringConnection_peerRegionAutoAccept(t *testing.T) {
+	var providers []*schema.Provider
```

Comment:
Else get errors like

```
    === CONT  TestAccAWSVPCPeeringConnection_peerRegionAutoAccept
        testing.go:677: Step 0, expected error:
    
            Error initializing context: 2 problems:
    
            - Could not satisfy plugin requirements:
            Plugin reinitialization required. Please run "terraform init".
    
            Plugins are external binaries that Terraform uses to access and manipulate
            resources. The configuration provided requires plugins which can't be located,
            don't satisfy the version constraints, or are otherwise incompatible.
    
            Terraform automatically discovers provider requirements from your
            configuration, including providers used in child modules. To see the
            requirements and constraints from each module, run "terraform providers".
    
            - provider "awsalternate" is not available
    
            To match:
    
            .*peer_region cannot be set whilst auto_accept is true when creating a vpc peering connection.*
    
    --- FAIL: TestAccAWSVPCPeeringConnection_peerRegionAutoAccept (0.87s)
```